app/help/en/index.md

```diff
@@ -1,9 +1,15 @@
-# Graphene Help
+# Help
 
-BitShares 2.0 is a Financial Smart Contracts platform that is based on the
-[Graphene](https://github.com/cryptonomex/graphene) technology built by
-[Cryptonomex](http://cryptonomex.com). You can see Graphene as a toolkit or
-technology for real-time blockchains.
+This is the reference wallet of BitShares Blockchain which connects to BitShares Blockchain nodes (bitshares-core).
+
+This reference wallet is a browser based wallet, which means your keys are stored locally in your browser and never leaves it while using the wallet. You should never expose your keys to anyone as transactions are signed locally before transmitting them to a node.
+
+BitShares Blockchain Reference wallet users can:
+
+- Login to BitShares Blockchain with their private keys to send and receive core cryptocurrency BTS.
+- Issue new tokens or utilize issued tokens at BitShares Blockchain.
+- Send, receive, burn/issue and freeze/unfreeze issued tokens at BitShares Blockchain.
```

Comment:
This list is not complete, should be formulated as such (or completely omitted)

Reply:
Please propose missing information.